response_operations_ui/templates/create-ce-event.html

```diff
@@ -86,15 +94,12 @@ <h2 id='error-saving-message' class="u-fs-r--b">Error creating {{event_name}} da
       {{ form.submit(class_="btn u-mb-s") }}
 
       <div>
-        <a href=" {{ url_for('collection_exercise_bp.view_collection_exercise', short_name=survey.shortName, period=period) }}" onclick="history.back(); return false;">Cancel</a>
```

Comment:
as far as I can tell the only thing that binding it to back was doing was causing some unintended effects when failing form validation and then trying to back out of the form, in which case it would just reload the last load of the form.

Reply:
Fair play, resolved.